### PR TITLE
[New practice exercise]: Two Bucket

### PR DESCRIPTION
**Pull request by ryanplusplus**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -677,6 +677,14 @@
         "prerequisites": [],
         "difficulty": 5
       },
+      {
+        "slug": "two-bucket",
+        "name": "Two Bucket",
+        "uuid": "20c471f8-e45e-4313-a23a-85d4a36c7185",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
```
Comment (meatball133) [inline config.json:686]: I think this exercise is quite hard and since it uses structs and enums I think perhaps the difficulty should be put as 6.

**Review by meatball133 — COMMENTED** [review]
Looks good!
Just one tiny comment

File: config.json (hunk 1 above)
Comment (ryanplusplus) [reply config.json]: Good thought, fixed!

**Review by meatball133 — APPROVED** [review]
Awesome!✨

**Pull request merged**